### PR TITLE
The 2024.12 release 🥳

### PR DESCRIPTION
**Pull request by mika**
Just in time before advent ends and we hit the border!

### CODE REVIEW

File: content/changelogs/README-grml-2024.12/index.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+**NOTE:** [20 Years of Grml Releases](https://blog.grml.org/archives/417-20-years-grml-releases.html), please join us in celebrating!
+
+### Important changes
+
+* We are proud to announce our first release supporting 64-bit ARM CPUs (Architecture **arm64**).
+* Please be aware that we [cannot support 32-bit x86 PCs](https://blog.grml.org/archives/416-grml32-sunset.html) anymore, and **no i386** ISO images have been produced.
+* New **Grml flavour naming schema**: now that the i386 flavour grml32 is gone, also grml96 no longer exists as such (grml96 was known as combination of grml64 + grml32 in one single ISO).
+  Instead we gained the new architecture known as `arm64`.
+  Due to this our new naming convention now is _grml-$FLAVOUR-$VERSION-$ARCH_.
+  For example _grml-full-2024.12-amd64_ reflects that it is the _grml-full_ flavour in version _2024.12_ for architecture _amd64_.
```
Comment (zeha) [inline content/changelogs/README-grml-2024.12/index.md:30]:
```suggestion
  For example _grml-full-2024.12-amd64_ reflects is the _grml-full_ flavour in version _2024.12_ for architecture _amd64_.
```

**Comment by zeha** [comment]
Please cherry-pick ce98bc27d68484a64366f02f0ccc75afe854d209 into this branch

**Review by zeha — APPROVED** [review]

**Comment by zeha** [comment]
> Please cherry-pick [ce98bc2](https://github.com/grml/grml.org/commit/ce98bc27d68484a64366f02f0ccc75afe854d209) into this branch

Just noticed the commit message in that commit is obviously wrong 🙈

**Comment by mika** [comment]
> > Please cherry-pick [ce98bc2](https://github.com/grml/grml.org/commit/ce98bc27d68484a64366f02f0ccc75afe854d209) into this branch
> 
> Just noticed the commit message in that commit is obviously wrong 🙈

no worries, fixed! :)

**Pull request merged**